angular/projects/public-nrpti/src/assets/styles/layout/layout.scss

```diff
@@ -211,29 +168,25 @@ main {
 
   @media screen and (min-width: 768px) {
     display: flex;
-    left: 250px;
     top: 69px;
+    left: 0;
     right: 0;
-
-    ol {
-      flex: 100%;
-      margin-bottom: 0;
-    }
+    padding: 0.65rem 1.25rem;
   }
 
-  @media screen and (max-width: 768px) {
-    position: relative;
-    top: 0;
-    padding-left: 45px;
+  // @media screen and (max-width: 1200px) {
```

Comment:
Just toss it entirely, no need to comment out

Reply:
Roger Roger